modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -42,4 +44,26 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
       '</td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table><p>x</p>'
     );
   });
+  it('TBA: Fire TableModified event when row are added via Tab key', () => {
```

Comment:
```suggestion
  it('TINY-7006: Fire TableModified event when rows are added via the Tab key', () => {
```

Reply:
Also please add a blank line between the tests.